.gitignore

```diff
@@ -116,3 +116,4 @@ run/
 
 # Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
 !gradle-wrapper.jar
+/CHANGELOG.md
```

Comment:
if this is a custom changelog pertaining to the mod (not t4j), i wouldn't mind it being tracked in the repo, but up to you

Reply:
I don't really see a reason to keep it there as it is just a file to make it easier to write the changelog for Modrinth